src/rulesets/Base/nondiff.jl

```diff
@@ -461,3 +461,11 @@ VERSION >= v"1.1" && @non_differentiable Sys.isopenbsd(::Symbol)
 @non_differentiable Threads.nthreads()
 @non_differentiable Threads.threadid()
 @non_differentiable Threads.threadid(::Task)
+
+@non_differentiable similar(::Any...)
+@non_differentiable eltype(::Type)
+
+@non_differentiable one(::Any)
+@non_differentiable ones(::Any...)
+@non_differentiable zero(::Any)
+@non_differentiable zeros(::Any...)
```

Comment:
```suggestion
@non_differentiable eltype(::Type)
```

could we also move the eltype to the right place alphabetically?

Reply:
Do you mean
```suggestion
@non_differentiable eltype(::Type)
@non_differentiable similar(::Any...)

@non_differentiable one(::Any)
@non_differentiable ones(::Any...)
@non_differentiable zero(::Any)
@non_differentiable zeros(::Any...)
```
?

Reply:
No, sorry, I did mean to delete the others, since they are already in base.jl on master, see e.g. https://github.com/JuliaDiff/ChainRules.jl/blob/3f18cd6944378f16827b0e47bc3aa4a264caed55/src/rulesets/Base/nondiff.jl#L307
```julia
julia> @which ones
Base
```

Reply:
actually eltype also seems to belong to Base

Reply:
Ah, sorry,  I misunderstood. `eltype` should still be in `Base/nondiff.jl` though, right?

Reply:
Correct! My bad, forgot that `Base/base.jl` exists 😄